spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaHealthCheckHandler.java

```diff
@@ -104,20 +115,39 @@ void populateHealthIndicators(Map<String, HealthIndicator> healthIndicators) {
 		}
 	}
 
+	void populateReactiveHealthIndicators(
+			Map<String, ReactiveHealthIndicator> reactiveHealthIndicators) {
+		for (Map.Entry<String, ReactiveHealthIndicator> entry : reactiveHealthIndicators
+				.entrySet()) {
+			this.reactiveHealthIndicators.put(entry.getKey(), entry.getValue());
+		}
+	}
+
 	@Override
 	public InstanceStatus getStatus(InstanceStatus instanceStatus) {
 		return getHealthStatus();
 	}
 
 	protected InstanceStatus getHealthStatus() {
-		Status status = getStatus(statusAggregator);
+		Status status = getStatus(this.statusAggregator);
```

Comment:
I am using the eclipse-code-formatter.xml from the Spring Cloud Build project, not sure if because of that saving the file in Eclipse automatically adds `this.` everywhere. Do you know if there's any way to fix this?

Reply:
Hmmm... this might be deprecated. We usually use IntelliJ. Will remove these manually.